includes/class-newspack-image-credits-settings.php

```diff
@@ -0,0 +1,193 @@
+<?php
+/**
+ * Settings Page.
+ *
+ * @package Newspack_Image_Credits
+ */
+class Newspack_Image_Credits_Settings {
+	/**
+	 * Set up hooks.
+	 */
+	public static function init() {
+		add_action( 'admin_init', [ __CLASS__, 'page_init' ] );
+		add_action( 'admin_enqueue_scripts', [ __CLASS__, 'enqueue_scripts' ] );
+	}
+
+	/**
+	 * Default values for site-wide settings.
+	 *
+	 * @return array Array of default settings.
+	 */
+	public static function get_default_settings() {
+		return [
+			[
+				'description' => __( 'A CSS class name to be applied to all image credit elements. Leave blank to display no class name.', 'newspack-image-credits' ),
+				'key'         => 'newspack_image_credits_class_name',
+				'label'       => __( 'Image Credit Class Name', 'newspack-image-credits' ),
+				'type'        => 'input',
+				'value'       => __( 'image-credit', 'newspack-image-credits' ),
+			],
+			[
+				'description' => __( 'A label to prefix all image credits. Leave blank to display no prefix.', 'newspack-image-credits' ),
+				'key'         => 'newspack_image_credits_prefix_label',
+				'label'       => __( 'Image Credit Label', 'newpack-listings' ),
```

Comment:
Need to change the domain from `newspack-listing` to `newspack-image-credits`

Reply:
Whoops! Updated in fcc3699